### ISSUE TITLE
Prebuilt binary crashing on RHEL 6

### ISSUE THREAD

**Issue by miken32** [doublemarket/grafana-rrd-server#37]
I've downloaded the prebuilt binary, and it keeps crashing with this error:

    grafana-rrd-server: symbol lookup error: grafana-rrd-server: undefined symbol: rrd_info_r

This happens every time it's sent a POST from Grafana.

It appears to be looking for an old version of librrd:

    root@host:~ ❱❱❱ objdump -x /usr/local/bin/grafana-rrd-server  | grep librrd
      NEEDED               librrd.so.4

`librrd.so.4` is present on my machine (version 1.3.8, installed by yum) but it does not have that symbol. I've also got `librrd.so.8` (version 1.6.0 installed from source) and it does have it:

    root@host:~ ❱❱❱ objdump -x /usr/lib64/librrd.so.4 | grep rrd_info_r
    root@host:~ ❱❱❱ objdump -x /usr/local/lib/librrd.so.8 | grep rrd_info_r
    0000000000030860 g     F .text	0000000000000c06              rrd_info_r
    root@host:~ ❱❱❱ 

Also checked RHEL 7; it also provides `librrd.so.4` version 1.4.8, and doesn't contain that symbol either.

**Comment by doublemarket** [comment]
I think this is caused by an [issue in the rrd package](https://github.com/ziutek/rrd/pull/20) which grafana-rrd-server is using. If this is correct, it should be fixed in the package's repository by merging the pull request (unlikely to happen, though 😭)

[The grafana-rrd-server binary is built](https://github.com/doublemarket/grafana-rrd-server/blob/3e319cf266c9819047622e3e57196a8e82d75098/.circleci/config.yml#L49) on a [Docker image](https://hub.docker.com/_/golang/), which is based on Debian. That is why it can be successfully built, but can't run on RHEL.

**Comment by doublemarket** [comment]
@miken32 👆

**Comment by miken32** [comment]
Yes that’s probably why it crashes, but the real problem is that it shouldn’t be looking for `librrd.so.4` because it doesn’t contain those symbols, but rather a later version. Then it would ignore the system version and insist on using manually built one instead.
